src/WebJobs.Script.WebHost/Security/Authorization/Policies/AuthorizationOptionsExtensions.cs

```diff
@@ -114,9 +80,14 @@ private static void AddScriptAuthenticationSchemes(this AuthorizationPolicyBuild
             builder.AuthenticationSchemes.Add(JwtBearerDefaults.AuthenticationScheme);
         }
 
+        /// <summary>
+        /// When AdminIsolation is enabled, performs platform internal request verification on the specified HTTP context.
+        /// </summary>
+        /// <param name="httpContext">The <see cref="HttpContext"/> to check.</param>
+        /// <param name="allowAppServiceInternal">True if App Service internal requests should also be allowed; otherwise, false.</param>
+        /// <returns>True if the request passes isolation checks, false otherwise.</returns>
```

Comment:
Following up on our offline conversation.

This method call is only required to ensure internal calls can bypass admin isolation. It would be good to have a clearer name to make sure this is easy to understand.

My suggestion is rename this method to something like `EnforceAdminIsolation` or something along those lines.

Reply:
Like the suggested name